src/group_theory/quotient_group.lean

```diff
@@ -80,14 +80,24 @@ instance : group (quotient N) :=
 @[to_additive quotient_add_group.mk' "The additive group homomorphism from `G` to `G/N`."]
 def mk' : G →* quotient N := monoid_hom.mk' (quotient_group.mk) (λ _ _, rfl)
 
+@[simp, to_additive quotient_add_group.eq_zero_iff]
+lemma eq_one_iff {N : subgroup G} [nN : N.normal] (x : G) : (x : quotient N) = 1 ↔ x ∈ N :=
+begin
+  refine quotient_group.eq.trans _,
+  rw [mul_one, subgroup.inv_mem_iff],
+end
+
 @[simp, to_additive quotient_add_group.ker_mk]
 lemma ker_mk :
   monoid_hom.ker (quotient_group.mk' N : G →* quotient_group.quotient N) = N :=
+subgroup.ext eq_one_iff
+
+@[to_additive quotient_add_group.eq_of_normal]
+lemma eq_of_normal {N : subgroup G} [nN : N.normal] {x y : G} :
+  (x : quotient N) = y ↔ x / y ∈ N :=
```

Comment:
Perhaps since the key feature of this lemma is `div`, this should be
```suggestion
@[to_additive quotient_add_group.eq_iff_sub_mem]
lemma eq_iff_div_mem {N : subgroup G} [nN : N.normal] {x y : G} :
  (x : quotient N) = y ↔ x / y ∈ N :=
```